lib/interactive.dart

```diff
@@ -499,6 +508,12 @@ class Interactive {
   }
 
   void _renderChart() {
+    _peopleCountWrapper.nodes.clear();
+    var peopleCountLabel = html.ParagraphElement()
+      ..text =
+          "Showing $_currentPeopleCount of $_totalPeopleCount people who has data for and fit the chosen criteria.";
```

Comment:
The phrasing of "who has data for" sounds a bit odd. But also, are there any people in list we don't have the data for?

Reply:
Yeah I thought I will add something basic and let Anna decide on the exact phrase. 

But yes a large number of people do not have any demographic data attached to them. In the filtered list, like when we choose show by age, I only show people who have age information available to them.